src/DocChecks.jl

```diff
@@ -232,14 +232,14 @@ function eval_repl(code, sandbox, meta::Dict, doc::Documents.Document, page)
     end
 end
 
-function eval_script(code, sandbox, meta::Dict, doc::Documents.Document, page)
+function eval_script(block, sandbox, meta::Dict, doc::Documents.Document, page)
     # TODO: decide whether to keep `# output` syntax for this. It's a bit ugly.
     #       Maybe use double blank lines, i.e.
     #
     #
     #       to mark `input`/`output` separation.
-    input, output = split(code, "\n# output\n", limit = 2)
-    result = Result(code, "", output, meta[:CurrentFile])
+    input, output = split(block.code, "\n# output\n\n", limit = 2)
```

Comment:
Any dangers with this? The problem was that `output` here became e.g. `"\n2" while the result string became "3", thus we replaced 
````
```jldoctest
1 + 2

# output

2
```
````
with 
````
```jldoctest
1 + 2

# output
3
```
````
instead of 
````
```jldoctest
1 + 2

# output

3
```
````

Reply:
Perhaps it would be safer to do
```julia
input, output = split(block.code, "# output\n", limit = 2)
input = rstrip(input, '\n')
output = lstrip(output, '\n')
```
?

Reply:
From the manual:
> Note that the amount of whitespace appearing above and below the # output line is not significant and can be increased or decreased if desired.

So we should strip `'\n'` instead